static/app/views/issueList/savedIssueSearches.spec.tsx

```diff
@@ -43,6 +43,7 @@ describe('SavedIssueSearches', function () {
     name: 'Last 4 Hours',
     query: 'age:-4h',
     visibility: SavedSearchVisibility.ORGANIZATION,
+    sort: 'date',
```

Comment:
What's this doing here?

Reply:
the form was invalid because this was missing in the initialData object, so to make the test pass, I had to add this value for the form to be complete. Does that make sense?

Reply:
ah got it